pygmt/tests/test_grdimage.py

```diff
@@ -206,9 +202,10 @@ def test_grdimage_central_meridians(grid, proj_type, lon0):
 # are still slightly different with an RMS error of 25, see issue at
 # https://github.com/GenericMappingTools/pygmt/issues/390
 # TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
+#       combine with test_grdimage_central_meridians_and_standard_parallels_fix
 @check_figures_equal(tol=1.5)
```

Comment:
Can we remove the `tol=1.5` with GMT 6.3?

Reply:
Yes, it's now removed.

Reply:
Err, I think the `@check_figures_equal` decorator needs to be kept 😅

Reply:
Wow, the dangers of 'just one more task' at the end of a workday 😅. I've added the decorator back in, so we can find out if the `tol` can be removed.